src/core/lib/iomgr/error.cc

```diff
@@ -454,7 +454,7 @@ typedef struct {
   grpc_status_code code;
   const char* msg;
 } special_error_status_map;
-static special_error_status_map error_status_map[] = {
+static const special_error_status_map error_status_map[] = {
```

Comment:
we should probably do these checks everywhere. I didn't realize that compilers could miss optimizing for this.

Reply:
Yes, we should definitely hunt for these fast path branches.